src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -566,13 +568,20 @@ export default class IconPainter {
     });
     canvas.setIconOriginToShapeBottomLeft();
 
-    const w = originalIconSize.width;
-    const h = originalIconSize.height;
+    // Loop
+    canvas.begin();
+    canvas.moveTo(5.5, 19.08);
+    canvas.arcTo(10, 10, 0, 1, 1, 10.5, 21.08);
+    canvas.fillAndStroke();
 
-    // Temporary render
-    c.setStrokeColor('Chartreuse');
-    canvas.roundrect(0, 0, w, h, 2, 2);
-    canvas.stroke();
+    // Arrow
+    canvas.setFillColor('Black');
```

Comment:
⚠️ we usually don't hard code color
We use the stroke color instead
We may also pass the color in the paint parameter

Reply:
Done 😄